torchmdnet/models/tensornet.py

```diff
@@ -265,10 +269,13 @@ def forward(
         x = torch.cat((tensor_norm(I), tensor_norm(A), tensor_norm(S)), dim=-1)
         x = self.out_norm(x)
         x = self.act(self.linear((x)))
+        v = skewtensor_to_vector(A)
+        v = v.transpose(1, 2)
```

Comment:
Would it not be possible to generate this tensor already transposed? By changing how the tensor is stacked.

Reply:
I explicitly asked him to build skewtensor_to_vector acting and giving outputs in a way we use vector_to_skewtensor and vector_to_symtensor, which implies having to transpose later on, it is in the issue.

Reply:
I remember checking what `TorchMD_ET` returns as v and its shaped like `(num_atoms, 3, hidden_channels)` whereas `A` is shaped like `(num_atoms, hidden_channels, 3, 3)`.

Reply:
Yes, I imagined. Linears act on the last dimension (they mix hidden_channels), and since for the ET the default output is EquivariantScalar, we could already take for granted that v will be used by linears. I would say, leave it in the way you prefer, because maybe we will have to revisit with @RaulPPelaez again? I have just realized (lol) that if the do everything from the beginning (3,3,hidden) we might be able to remove those ugly permutes from the tensor linears.

Reply:
yes exactly. My only problem with this to get it merged is that the vector features should not be generated by default.
An option to TensorNet is not ideal, but I do not see another way out of it...

Reply:
then let's add the flag: vector_output

Reply:
Just updated it with `vector_output` flag.